### PR TITLE
[WIP] UI Improvements

### LINKED ISSUE #131: Big Neon Native App UI Improvements
This task is broken down into subtasks for each screen/feature.

Please note: In the Invision screens the last screen(s) for each section is the current app so its easy to compare. Also for developers: with Invision you can use the inspect feature (shortcut "i") to get the design specs. 

https://tari-labs.invisionapp.com/overview/Big-Neon---App-UI-Updates-cjzbz2m620k02012hkkbsoca1/screens?v=g22Bo6rvonmzW4dXGWr7qw%3D%3D&linkshare=urlcopied 
Big Neon app review + screens suggestions.pdf

### PR DESCRIPTION
**Pull request by mrmaDrma**
Resolves #131 

Still waiting on web part for payment view

### CODE REVIEW

**Comment by Krakaw** [comment]
@mrmaDrma please merge develop into this branch, solve the conflicts, and post a screenshot / cloudapp video.

File: src/Events/show.js [hunk 1]
```diff
@@ -1,472 +1,440 @@
...
+            ref={(ref) => (this.scrollView = ref)}
+            keyboardShouldPersistTaps="handled"
+            showsVerticalScrollIndicator={false}
+          >
+            {this.renderScreen()}
+          </ScrollView>
+        </KeyboardAvoidingView>
+        {this.renderBackArrow()}
+        {this.renderTickets()}
+        {/* {this.renderPurchaseTicketButton()} */}
```
Comment (mrmaDrma) [inline src/Events/show.js]: @Krakaw I'm not really sure why this button is here. We can talk about reimplementation and moving of this on payment view?

**Comment by coka** [comment]
@Krakaw Can we close this?

**Comment by Krakaw** [comment]
Let's close, things have changed.

**Pull request closed without merging**